tests/emerald/emerald_test.go

```diff
@@ -0,0 +1,192 @@
+package emerald
```

Comment:
This will work for any runtime with the accounts module (= all of them, today), so maybe this can be `runtime.go`?

Also, this looks like an opportunity to do some long-overdue naming cleanup around these tests. The "genesis" test is not really testing the genesis, it's comparing against a state dump at a given non-genesis height. The dump is called "genesis" for historical reasons on the oasis-core side, but I've always found it super misleading here.
Also, I feel that emerald_test.go (from this PR) and genesis_test.go have more in common than any two other tests, and the commonality is stronger than the distinction that one works on consensus and one on runtimes. They are both pretty special in how much ops setup they require, and how ultra slow they are. They arguably both abuse the testing framework slightly. 

So WDYT about the following reorg?
```
git mv tests/genesis tests/statecheck
git mv tests/statecheck/{genesis_test,consensus}.go
git mv tests/emerald/emerald_test.go tests/statecheck/runtime.go
rmdir tests/emerald
```

Reply:
Agreed, updated!